addons/account/static/src/components/duplicate_bill_warninng/duplicate_bill_warning.js

```diff
@@ -0,0 +1,67 @@
+/** @odoo-module **/
+
+import { registry } from "@web/core/registry";
+import { useService } from "@web/core/utils/hooks";
+import { Component } from "@odoo/owl";
+import { getCurrency } from "@web/core/currency"
+
+
+class DuplicateBillWarning extends Component {
+    static template = "account.DuplicateBillWarning";
+    static props = {};
+    setup() {
+        this.action = useService("action");
+        this.orm = useService("orm");
+    }
+
+    async discard() {
+        if (this.record?.isInEdition) {
+            await this.record.discard();
+        }
+    }
+
+    async open_bills_tree_and_discard() {
+        await this.discard();
+        const action = await this.orm.call("account.move", "open_duplicated_ref_bill_view", [this.props.record.resId], {});
```

Comment:
Do you need to do an rpc call?
I feel like this is just the default action with the domain that is constrained on `id` (so basically the same as `open_bill_form_and_discard` only with the `list` view added, and the domain)

Reply:
i did this to keep the diff minimal because if I don't use the `open_duplicated_ref_bill_view` action it would be unusued and would have to remove it I guess

Reply:
Better to remove code than to do useless rpc call :laughing: 
Also, it contributes to making this widget agnostic of the model since you would be able to use `res_model` (if needed in the future we can add props to select the related action, no need for now)